docs/Dockerfile

```diff
@@ -13,18 +13,15 @@ RUN groupadd jenkins --gid=$GID -o && useradd jenkins --uid=$UID --gid=$GID --cr
 RUN (mkdir -p /var/lib/teleport && chown -R jenkins /var/lib/teleport)
 
 # Install SASS
-RUN (apt-key update && \
-             apt-get -q -y update --fix-missing && \
-             apt-get -q -y update && \
-             apt-get install -q -y ruby-sass)
-             
-ENV LANGUAGE="en_US.UTF-8" \
-    LANG="en_US.UTF-8" \
-    LC_ALL="en_US.UTF-8" \
-    LC_CTYPE="en_US.UTF-8" \
+RUN apt-get update -y && apt-get install -q -y ruby-sass parallel
```

Comment:
parallel isn't needed anymore, with the death of build.sh

Reply:
Same here - if it's not needed anymore - remove it.

Reply:
removed in b4b64f6.  I was able to go a bit further and remove sass (and thus all the apt directives) in 7528d55.